static/sitemap.xml

```diff
@@ -93,7 +93,7 @@
       </url>
       <url>
           <loc>https://www.operationcode.org/podcast</loc>
-          <lastmod>Sun Jul 14 2019 14:53:00 GMT+0530 (India Standard Time)</lastmod>
+          <lastmod>Mon Jul 15 2019 21:14:41 GMT+0530 (India Standard Time)</lastmod>
```

Comment:
Last comment! It looks like time zone is being taken from the user's local filesystem. To avoid diffs, it'd be cool if we could set the timezone (maybe GMT 0 or GMT+8 [USA PDT]), so that no matter who edits the filesystem, the time stamps on all the pages wont change.

Reply:
@kylemh . It is done. I hope toUTCString() function would work as expected in all timezones.